db/migrate/20220909190221_drop_cockpit_role.rb

```diff
@@ -0,0 +1,15 @@
+class DropCockpitRole < ActiveRecord::Migration[6.0]
+  class SettingsChange < ActiveRecord::Base; end
+
+  def up
+    say_with_time("Deleting cockpit role configuration") do
+      SettingsChange.where("key = '/server/role' AND value LIKE '%ws_cockpit%'").select('id', 'value').each do |row|
+        if row.value == 'ws_cockpit'
+          row.delete
+        elsif row.value.include?("ws_cockpi")
```

Comment:
```suggestion
        elsif row.value.include?("ws_cockpit")
```

Reply:
Although I don't thin this check is necessary because you check it at the query level with the `LIKE` clause.

Reply:
wonder why this was green with this bug.